src/main/java/com/github/manolo8/darkbot/utils/DiscordWebhook.java

```diff
@@ -0,0 +1,339 @@
+                EmbedObject.Thumbnail thumbnail = embed.getThumbnail();
+                EmbedObject.Author author = embed.getAuthor();
+                List<EmbedObject.Field> fields = embed.getFields();
+
+                if (footer != null) {
+                    JsonObject jsonFooter = new JsonObject();
+
+                    jsonFooter.addProperty("text", footer.getText());
+                    jsonFooter.addProperty("icon_url", footer.getIconUrl());
+                    jsonEmbed.add("footer", jsonFooter);
+                }
+
+                if (image != null) {
+                    JsonObject jsonImage = new JsonObject();
+
+                    jsonImage.addProperty("url", image.getUrl());
+                    jsonEmbed.add("image", jsonImage);
+                }
+
+                if (thumbnail != null) {
+                    JsonObject jsonThumbnail = new JsonObject();
+
+                    jsonThumbnail.addProperty("url", thumbnail.getUrl());
+                    jsonEmbed.add("thumbnail", jsonThumbnail);
+                }
+
+                if (author != null) {
+                    JsonObject jsonAuthor = new JsonObject();
+
+                    jsonAuthor.addProperty("name", author.getName());
+                    jsonAuthor.addProperty("url", author.getUrl());
+                    jsonAuthor.addProperty("icon_url", author.getIconUrl());
+                    jsonEmbed.add("author", jsonAuthor);
+                }
+
+                List<JsonObject> jsonFields = new ArrayList<>();
+                for (EmbedObject.Field field : fields) {
+                    JsonObject jsonField = new JsonObject();
+
+                    jsonField.addProperty("name", field.getName());
+                    jsonField.addProperty("value", field.getValue());
+                    jsonField.addProperty("inline", field.isInline());
+
+                    jsonFields.add(jsonField);
+                }
+//                jsonEmbed.add("fields", jsonFields.toArray());
+                jsonFields.forEach(jsonField -> jsonEmbed.add("fields", jsonField));
+                embedObjects.add(jsonEmbed);
+            }
+//            json.add("embeds", embedObjects.toArray());
+            embedObjects.forEach(embedObject -> json.add("embeds", embedObject));
+        }
+
+        URL url = new URL(this.url);
+        HttpsURLConnection connection = (HttpsURLConnection) url.openConnection();
+        connection.addRequestProperty("Content-Type", "application/json");
+        connection.addRequestProperty("User-Agent", "Java-DiscordWebhook-BY-Gelox_");
+        connection.setDoOutput(true);
+        connection.setRequestMethod("POST");
+
+        OutputStream stream = connection.getOutputStream();
+        stream.write(json.toString().getBytes());
+        stream.flush();
+        stream.close();
+
+        connection.getInputStream().close(); //I'm not sure why but it doesn't work without getting the InputStream
+        connection.disconnect();
```

Comment:
use our `Http` library class for this instead, should be just a couple linese using it